test/libyul/ewasmTranslationTests/blockhash.yul

```diff
@@ -3,6 +3,5 @@
 }
 // ----
 // Trace:
-//   INVALID()
```

Comment:
So `blockhash(1)` returns 0, because it is not found. Should add another statement too which actually returns a blockhash.

Reply:
@rodiazet to clarify: could you add another stetement (or test file) which has a successful call to blockhash?

Reply:
done